sdk/template/template/rollup.base.config.js

```diff
@@ -58,13 +58,12 @@ export function nodeConfig(test = false) {
 export function browserConfig(test = false, production = false) {
   const baseConfig = {
     input: input,
-    external: ["@azure/ms-rest-js"],
```

Comment:
@bterlson: This change is required for the browser tests to pass, since the global was renamed to `Azure.Core.HTTP` which causes a null ref if the browser tests are run without this dependency.  While the tests passed before, I think they were somewhat invalid and just happened to pass without the dependency.

This also aligns `template` with the existing rollup config used by `keyvault`.

The other option would be keeping this external but including the dependency in the karma config.

Reply:
I think you still need @azure/core-http as external, but otherwise this looks fine?

Reply:
@bterlson: Based on our offline discussion about bundling, are we now in agreement that `template` **should** include `core-http` in its browser bundle (meaning no further changes to this PR)?

Reply:
Yes let's include dependencies in browser bundles. Note that this means ujsers using multiple services should bundle their applications in order to avoid having multiple shared dependencies present. I'm not sure if this needs to be documented, but it can't hurt! Not sure the best place to put it though.